BootloaderCorePkg/Tools/BuildUtility.py

```diff
@@ -480,7 +480,8 @@ def gen_hash_file (src_path, hash_path = '', is_key = False):
 			raise Exception ("Invalid public key binary!")
 		di = di[4:]
 		di = di[:0x100][::-1] + di[0x100:][::-1]
-	ho = hashlib.sha256(di)
+	if hash_type == 'SHA2_256':
```

Comment:
what if hash_type is not SHA2_256 ?  Need an exception here.

Reply:
will add exception cases where ever its applicable.

Reply:
will do